packages/elements/src/components/ui/cards/CardHeader.tsx

```diff
@@ -18,27 +21,23 @@ export interface CardHeaderProps {
 export const CardHeader: React.VFC<CardHeaderProps> = ({
   text,
   variant = "standard",
+  leftIcon,
   contentAfterHeading,
   contentRight,
   contentLeft,
   contentCenter,
 }) => {
   return (
     <div className={cx(styles.cardHeader, styles[variant])}>
-      <Row alignItems={"center"}>
-        {contentLeft && (
-          <>
-            {contentLeft}
-            <Indent />
-          </>
+      <Row alignItems={"center"} gap={variant === "compact" ? 1 : 2}>
+        {contentLeft}
+        {leftIcon && (
+          <Icon icon={leftIcon} size={variant === "compact" ? 16 : 24} />
```

Comment:
Note for self: In general, we should use exhausting switch for these, so that new cases causes compile error.